source/whats-new.txt

```diff
@@ -73,6 +74,54 @@ New versions of the {+driver-short+} can introduce the following types of breaki
 
 .. _upcoming-breaking-changes:
 
+.. _version-3.11:
+
+What's New in 3.11
+------------------
+
+.. warning:: Build System Breaking Changes
+
+   The v3.11 driver introduces the following breaking changes to the build
+   system:
+   
+   - Drops support for {+mdb-server+} v3.6.
+   - Removes export of the following private member functions in the bsoncxx
+     ABI:
+   
+     - ``bsoncxx::v_noabi::types::bson_value::value::value(const uint8_t*, uint32_t, uint32_t, uint32_t)``
+     - ``bsoncxx::v_noabi::types::bson_value::view::_init(void*)``
+     - ``bsoncxx::v_noabi::types::bson_value::view::view(const uint8_t*, uint32_t, uint32_t, uint32_t)``
+     - ``bsoncxx::v_noabi::types::bson_value::view::view(void*)``
+
+   - Removes export of the following private member functions in the mongocxx
+     ABI:
+   
+     - ``mongocxx::v_noabi::options::change_stream::as_bson()``
+     - ``mongocxx::v_noabi::options::aggregate::append(bsoncxx::v_noabi::builder::basic::document&)``
+     - ``mongocxx::v_noabi::options::index::storage_options()``
+   
+   - Declare all exported functions symbols with ``__cdecl`` when compiled with
+     MSVC. This is an ABI breaking change if you use an alternative
+     default calling convention when building projects.
+
+The v3.11 driver release includes the following new features:
+
+- Adds support for {+mdb-server+} v8.0.
+- Adds support for Queryable Encryption range queries. To use this
+  feature, your app must connect to {+mdb-server+} 8.0 or later. For
+  more information about Queryable Encryption, see :manual:`Queryable
+  Encryption </core/queryable-encryption>` in the {+mdb-server+} manual.
+- Adds ``empty()`` member function for ``mongocxx::v_noabi::bulk_write``.
+- The minimum C driver version is now 1.28.0.
```

Comment:
Suggest moving to backwards breaking changes. Raising the minimum required C driver is also listed as a backwards breaking change in the v3.9 notes.